client/reducers/urlReducers.js

```diff
@@ -0,0 +1,19 @@
+import * as constants from '../actions/constants';
+import initialState from './initialState';
+
+/**
+ * Reducer for setting redirectUrl
+ * @param {string} [state=initialState.access]
+ * @param {object} action
+ * @returns {object} redirectUrl state
+ */
+function url(state = initialState.redirectUrl, action) {
```

Comment:
Also I feel we should adhere to ES6 function definition and not interchange them.

Reply:
COmment goes for the above file and viceversa.

Reply:
Umm... I think that is one correct way of defining functions even in ES6. Correct me if I am wrong

Reply:
I still think this is a valid way of defining ES6 function. We can let ESLint decide otherwise when we set up ESLint with ES6 settings.